### ISSUE TITLE
Unplugging/Plugging eGPU causing video wallpaper to silently crash

### ISSUE THREAD

**Issue by HunterAhlquist** [rocksdanister/lively#1938]
**Describe the bug**
When plugging in or unplugging my eGPU, the wallpaper engine crashes. I have tried to stop this by forcing the program to use my iGPU because that never stops operation.

**To Reproduce**
1. Plug in eGPU
2. Start video wallpaper
3. Unplug eGPU
4. The video wallpaper should stop/crash/end with no apparent error on the user-end

**Expected behavior**
The video wallpaper continues playing, recovers, or restarts.

**Screenshots/Video**
N/A

**Desktop (please complete the following information):**
 - OS: Windows 11 22H2 Build, 22621.2428
 - Video wallpaper
 - Thunderbolt 4 eGPU (Razer Chroma, 3070)

**Additional context**
~~The application itself doesn't crash, only the engine driving the wallpaper. Reselecting the wallpaper resets it with no app or computer restarts needed.~~ It actually does require an app restart, must have been a fluke that one time.

**Log file (Important)**
[lively_log_20231015_135630.zip](https://github.com/rocksdanister/lively/files/12910658/lively_log_20231015_135630.zip)

**Comment by rocksdanister** [comment]
Does it still crash with `GPU Decode` disabled? (Settings -> Wallpaper -> Video Player.)

**Comment by HunterAhlquist** [comment]
> Does it still crash with `GPU Decode` disabled? (Settings -> Wallpaper -> Video Player.)

Yes it does, I also tried just unplugging a monitor and it happens too, I have a 2 monitor setup. Seems it's not related to disconnecting a display adapter.
Also, it's now requiring me to restart the app before working again.

**Comment by NaeemBolchhi** [comment]
If I may add to this, I am experiencing the same bug, except it happens whenever I run any fullscreen game. I've tried Skyrim and Cricket 07 before writing this, and produces the same issue. Wallpaper pauses and won't start again till I have closed and restarted Lively. I tried disabling GPU Decode as mentioned here, but it still happens.

**Comment by rocksdanister** [comment]
Running fullscreen has this issue on Windows 22H2 https://github.com/rocksdanister/lively/issues/1926

**Comment by rocksdanister** [comment]
Can you verify with the latest v2.0.7.4 if the issue is fixed?
